packages/components/src/Menu/Menu.mdx

```diff
@@ -0,0 +1,75 @@
+---
+name: Menu
+menu: Components
+route: /components/menu
+---
+
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Menu } from ".";
+
+# Menu
+
+<ComponentStatus stage="pre" responsive="no" accessible="yes" />
+
+A toggleable menu that holds a list of actions.
```

Comment:
It might be nice to add a little line here as to when this should be used vs a select.